src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx

```diff
@@ -265,7 +265,7 @@ export default function QueryBarTopRow(props: QueryBarTopRowProps) {
 
     return (
       <NoDataPopover storage={storage} showNoDataPopover={props.indicateNoData}>
-        <EuiFlexGroup responsive={false} gutterSize="s" alignItems="flexStart">
+        <EuiFlexGroup responsive={false} gutterSize="none" justifyContent="flexStart">
```

Comment:
this gutter removal didn't regress anything?

Reply:
this is to be consistent with existing date picker in discover. But let me check the UX mock up
![image](https://github.com/user-attachments/assets/1e20e3e1-cc58-47d1-995a-1ac1ff5bc329)

Reply:
revert this back, also update discover header to use gutter size "s", based on UX mock, there should be spacing between refresh button and datepicker